docs/app/pages/components/sections/input-controls/sliders/sliders.component.html

```diff
@@ -77,9 +77,7 @@ <h4 class="text-black p-t-md p-b-md">Range Slider with Text Inputs</h4>
 <p>The slider component can be used to allow a user to select a single value or a range of values. It can easily be customized
     to match the desired appearance and behavior.</p>
 
-<br>
-
-<h4>Slider Component</h4>
+<h4 class="m-t-lg">Slider Component</h4>
```

Comment:
I meant to add a rule like:

.uxd-component-section-content > * > h4 { ... }

Then all documentation sections are consistent.